packages/cadl-python/src/emitter.ts

```diff
@@ -1256,7 +1278,8 @@ function getNamespaces(context: DpgContext): Set<string> {
 
 function emitCodeModel(context: EmitContext<EmitterOptions>) {
     const dpgContext = createDpgContext(context);
-    const clientNamespaceString = getClientNamespaceString(dpgContext);
+    let clientNamespaceString = getClientNamespaceString(dpgContext);
+    if (clientNamespaceString) clientNamespaceString = clientNamespaceString.toLowerCase();
```

Comment:
fix the comparation at https://github.com/Azure/autorest.python/blob/fbcadb4c739d4b7c8edab21a0fa66791b1bd40c2/packages/cadl-python/src/emitter.ts#L1266, since our namespaces are low cases (https://github.com/Azure/autorest.python/blob/fbcadb4c739d4b7c8edab21a0fa66791b1bd40c2/packages/cadl-python/src/emitter.ts#L1244)